### PR TITLE
Update SignalR, TS, and Webpack to 6.0

### LINKED ISSUE #24650: .NET 6: SignalR with TypeScript and Webpack
remove versions, ie use `npm i -D clean-webpack-plugin css-loader html-webpack-plugin mini-css-extract-plugin ts-loader typescript webpack webpack-cli`
Replace cook book instructions with copy the `src` folder from the download sample. Explain the contents of each file.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 47557f89-3a14-1139-0074-38d931b35faf
* Version Independent ID: 1cee43b4-b672-e480-3e26-72e08a6c2d77
* Content: [Use ASP.NET Core SignalR with TypeScript and Webpack](https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr-typescript-webpack?view=aspnetcore-6.0&tabs=visual-studio)
* Content Source: [aspnetcore/tutorials/signalr-typescript-webpack.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/tutorials/signalr-typescript-webpack.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-tutorials**
* GitHub Login: @ssougnez
* Microsoft Alias: **bradyg**

### PR DESCRIPTION
**Pull request by serpent5**
Fixes #24650.

[Internal Review](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/signalr-typescript-webpack?view=aspnetcore-6.0&branch=pr-en-us-25297&tabs=visual-studio).

### CODE REVIEW

**Comment by serpent5** [comment]
@Rick-Anderson I've linked through to `AspNetCore.Docs.Samples` for the sample, as you said to in the linked issue. It's all on a branch of mine, so I'll update that over to main and get it committed/merged once we get approval on this PR here.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**
